<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdMainWindow</class>
 <widget class="QMainWindow" name="CmdMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CmdMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_open">
      <property name="text">
       <string>open com and build gencmd</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_fillmaps">
      <property name="text">
       <string>fill maps test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_16Read">
         <property name="text">
          <string>16 bit read test</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_16Write">
         <property name="text">
          <string>16 bit write test</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_16r"/>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="lineEdit_16w"/>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_32w"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_32r"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_32Read">
         <property name="text">
          <string>32 bit read test</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_32Write">
         <property name="text">
          <string>32 bit write test</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_16w">
         <property name="maximum">
          <double>4294967296.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_32w">
         <property name="maximum">
          <double>4294967296.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
